tests/test_responses.py

```diff
@@ -309,3 +324,8 @@ def test_head_method(test_client_factory):
     client = test_client_factory(app)
     response = client.head("/")
     assert response.text == ""
+
+
+def test_empty_response():
```

Comment:
This is the test case that we should probably expand to cover the 6 different cases described. I guess we want these test cases?...

* `test_empty_response`
* `test_non_empty_response`
* `test_file_response_unknown_size`
* `test_file_response_known_size`
* `test_streaming_response_unknown_size`
* `test_streaming_response_known_size`

Reply:
When do we have a `FileResponse` with unknown size?